### ISSUE TITLE
keras_lstm.py does not show the results claimed in the tutorial?

### ISSUE THREAD

**Issue by dsmic** [adventuresinML/adventures-in-ml-code#14]
I run 

python keras_lstm.py 1 

with:
detlef@ubuntu-i7:~$ python -c 'import tensorflow as tf; print(tf.__version__)'  # for Python 2
1.12.0

(tested it with turned off gpu too but use cuda 9.1 on GTX1070Ti)

run 50 epoches and got only:
Epoch 50/50
1549/1549 [==============================] - 169s 109ms/step - loss: 5.2143 - categorical_accuracy: 0.1464 - val_loss: 7.6633 - val_categorical_accuracy: 0.0466

The tutorial says, you reach 40% training and 20% validation?

Thanks a lot for the wonderful tutorial!

**Comment by leigao97** [comment]
Same issue. Have you fixed this?

**Comment by dsmic** [comment]
No, sorry.
